src/graphql/graphql.schema.ts

```diff
@@ -251,7 +252,7 @@ const schema = gql`
   type Meta {
     description: String
     themes: [String!]!
-    datasetIds: [Int]
+    datasetIds: [Int!]
```

Comment:
@royderks Question, what is the intended use of this field? I can't find it being used anywhere in the code, perhaps we can get rid of it?

Reply:
We've added this to (in the future) link maplayers to their corresponding dcat dataset(s). Not use when we' get to this, so can be deleted :)

Reply:
Ok, I'll leave it for now as it does no harm.